### ISSUE TITLE
Using cassie.types.Long in find results in invalid query

### ISSUE THREAD

**Issue by sevenbitbyte** [Flux159/cassie-odm#3]
Using Model.find with an argument of type `cassie.type.Long` is incorrectly parsed and results in an invalid query string.

Example model:

```
cassie.model(
  'longtest',
  new cassie.Schema({
    field1 : {type: cassie.types.Long, index: true},
    field2: cassie.types.uuid
  });
);
```

Example Model.find

```
var query = cassie.model('longtest').find({ field1 : cassie.types.Long.fromString('12345678901234567890') });

console.log(query);
```

Produces the query:
```
{ 
  queryString: 'SELECT * FROM longtest WHERE  ',
  params: [ ],
  toString: [Function],
  limit: [Function],
  sort: [Function],
  page: [Function],
  exec: [Function],
  stream: [Function] }
```

The query string should be valid andthe params array should contain the supplied long:

```
queryString: 'SELECT * FROM longtest WHERE field1=? '
```

**Comment by sevenbitbyte** [comment]
Another way to repro is calling the argument parser directly

```
parseArguments({field1: { low: 964128835, high: 28, unsigned: undefined }}, 'primaryKey')
```

**Comment by Flux159** [comment]
Fixed via #4; Version tag for fix is v1.1.6.
